Bio/GenBank/__init__.py

```diff
@@ -654,14 +662,14 @@ def size(self, content):
         """Record the sequence length."""
         self._expected_size = int(content)
 
-    def residue_type(self, type):
+    def residue_type(self, res_type):
```

Comment:
Technically this is a change to the method's API, which is why I'd left it as is - but this seems unlikely to cause any real trouble.